design-proposals/volume-migration/volume-migration.md

```diff
@@ -0,0 +1,297 @@
+# Overview
+This design proposal extends KubeVirt APIs to enable the storage migration between PVCs.
+
+## Motivation
+In some circumstances, the guest owner may want to have the option of switching the storage provider or class for a group of volumes used by an active VM. The VM must currently be turned off and recreated for this operation causing service disruptions and downtime.
+
+
+## Goals
+Design and develop an API to handle storage migration between PVCs.
+
+ ## Non  Goals
+*  This features is not directly aimed for live migrating the VM using RWO volumes. Hence, the API covers and refers only to the volume migration.
+*  This design doesn't cover migration between storage classes, but rather implements a basic APIs that can be used by overlaying tooling
+*  This design doesn't include the creation of the destination PVCs, but it expects the destination PVCs to exist.
```

Comment:
remove `but` so it reads `the destination PVCs, it expects the destination PVCs to exist`